src/schema.toml

```diff
@@ -59,17 +59,18 @@ images.landing_rooms_thumb = { category = "images", type = "file", internal = "t
-theme.action-label-color = { category = "theme", type = "color", name = "Primary Label Color", description = "Call to action label color.", deprecated="true" }
-theme.notice-background-color = { category = "theme", type = "color", name = "Secondary Color", description = "Notice and indicator color.", deprecated="true" }
-theme.notice-text-color = { category = "theme", type = "color", name = "Secondary Text Color", description = "Notice and indicator text color.", deprecated="true" }
-theme.notice-widget-color = { category = "theme", type = "color", name = "Secondary Widget Color", description = "Notice and indicator UI widget color.", deprecated="true" }
-theme.favorited-color = { category = "theme", type = "color", name = "Favorited Color", description = "Favorited Color" }
+theme.dark-theme = { category = "theme", type = "boolean", name = "Dark Mode", description = "[deprecated] Use dark UI styling.", deprecated="true" }
+theme.action-color = { category = "theme", type = "color", name = "Primary Color", description = "[deprecated] Call to action button color.", deprecated="true" }
+theme.action-color-highlight = { category = "theme", type = "color", name = "Highlighted Primary Color", description = "[deprecated] Highlighted call to action color.", deprecated="true" }
+theme.action-color-disabled = { category = "theme", type = "color", name = "Disabled Primary Color", description = "[deprecated] Disabled call to action color.", deprecated="true" }
+theme.action-text-color = { category = "theme", type = "color", name = "Primary Text Color", description = "[deprecated] Call to action button text color.", deprecated="true" }
+theme.action-subtitle-color = { category = "theme", type = "color", name = "Primary Subtitle Color", description = "[deprecated] Call to action button subtitle text color.", deprecated="true" }
+theme.action-label-color = { category = "theme", type = "color", name = "Primary Label Color", description = "[deprecated] Call to action label color.", deprecated="true" }
+theme.notice-background-color = { category = "theme", type = "color", name = "Secondary Color", description = "[deprecated] Notice and indicator color.", deprecated="true" }
+theme.notice-text-color = { category = "theme", type = "color", name = "Secondary Text Color", description = "[deprecated] Notice and indicator text color.", deprecated="true" }
+theme.notice-widget-color = { category = "theme", type = "color", name = "Secondary Widget Color", description = "[deprecated] Notice and indicator UI widget color.", deprecated="true" }
+theme.favorited-color = { category = "theme", type = "color", name = "Favorited Color", description = "[deprecated] Favorited Color", deprecated = "true" }
+theme.themes = { category = "theme", type = "longstring", name = "Themes", description = "A json array describing custom client themes.", json="true"}
```

Comment:
nit: I wonder if we should be using `type="json"` here instead of `json="true"`.